### ISSUE TITLE
ots says something went wrong when starting a task with no project

### ISSUE THREAD

**Issue by aisopuro** [hegenator/ots#10]
Ran the following:
```
$ ots start T11800
Something went wrong when trying to update data from Odoo.
'bool' object is not iterable
Timesheet stopped: T11550
Timesheet started: T11800
```
I suspect this was caused by the task not having a project set. The task `T11800` did not have a project set when I started it, which was by design (I did not know what the appropriate project was).

I think the automatic details update should not assume that `project_id` is set on tasks.

**Comment by aisopuro** [comment]
If we feel that some warning should be displayed, I think it should be a more explicit "heads up there's no project on this task", rather than this exception.

**Comment by hegenator** [comment]
Yea, I had always assumed that if you have a task, it belongs to a project (in Odoo). Apparently that is not the case. Should be an easy fix

**Comment by aisopuro** [comment]
Right, I just learned (or was reminded) that timesheet entries _do_ require a project. So it does seem very sensible to warn the user about the fact that they won't be able to push timesheet entries that don't have projects.
